app/models/manageiq/providers/openshift/inventory/persister/target_collection.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::Openshift::Inventory::Persister::TargetCollection < ManageIQ::Providers::Openshift::Inventory::Persister::ContainerManager
+  def targeted
+    false # TODO(lsmola) get ready for true
+  end
+
+  def strategy
+    nil
```

Comment:
same q as on kubernetes PR — is this method used?

Reply:
hm, right, I am changing it below, this can go away